src/Serilog.Sinks.SignalR/LoggerConfigurationSignalRExtensions.cs

```diff
@@ -26,30 +26,70 @@ namespace Serilog
     public static class LoggerConfigurationSignalRExtensions
     {
         /// <summary>
-        /// Adds a sink that writes log events as documents to a SignalR hub. 
+        /// Adds a sink that writes log events as documents to a SignalR server. 
         /// </summary>
         /// <param name="loggerConfiguration">The logger configuration.</param>
         /// <param name="context">The hub context.</param>
         /// <param name="restrictedToMinimumLevel">The minimum log event level required in order to write an event to the sink.</param>
         /// <param name="batchPostingLimit">The maximum number of events to post in a single batch.</param>
         /// <param name="period">The time to wait between checking for event batches.</param>
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
+        /// <param name="groupNames">Names of the Signalr groups you are broadcasting the log event to. Default is All Groups.</param>
+        /// <param name="userIds">ID's of the Signalr Users you are broadcasting the log event to. Default is All Users.</param>
+        /// <param name="excludedConnectionIds">Signalr connection ID's to exclude from broadcast.</param>
         /// <returns>Logger configuration, allowing configuration to continue.</returns>
         /// <exception cref="ArgumentNullException">A required parameter is null.</exception>
-        public static LoggerConfiguration SignalR(
+        public static LoggerConfiguration SignalRServer(
```

Comment:
Sorry, I meant potentially renaming the _new_ method, not the existing one. I think we should keep `WriteTo.SignalR()` since it'd be an annoying breaking change to deal with otherwise.

It is tough then to name the new method; maybe `SignalRClient()`? It's a tricky one... :-)